### ISSUE TITLE
Selector to filter whole group by region (and some meta about development)

### ISSUE THREAD

**Issue by alexupotsdam** [TelemetryDeck/Viewer#133]
I have set up my groups but if I want to focus on a country, I would have to edit each insight to filter by that. Considering how jumpy editing the insights is, that would 

1) take a lot of time
2) be a weird thing to constantly be doing in there

A small bar with a country selection (favorites+all) at the top of each group would make this much more useful.

_Some meta about development:_

I honestly think you should open-source the viewer. The clunkiness of the viewer right now is a big issue IMO and I could and would happily implement features like my suggestion within a few hours.

I really enjoy AT so far but I think at the moment you are at a weird spot in regards to find funding and creating a sustainable business. If the viewer is your business model that puts you in a weird position how you value the company. Open-sourcing the viewer - under reserve that people actually contribute - may get you more people signed up and I think client count is what you could base your company's value around in future. 

Just my two cents, I see a lot of potential if you are able to establish the seemingly first "privacy no brainer" on great scale, it's what a lot of small devs need.

**Comment by winsmith** [comment]
Yeah I've been playing with that idea as well. The viewer is not what generates value, our stack is. Lets see what the future holds, it seems very likely we're going in that direction.

**Comment by winsmith** [comment]
In the meantime, I think you could create a similar effect by having separate insight groups for separate locales. This way you can switch them quickly.
